egs/thchs30/s5/local/dae/train.sh

```diff
@@ -0,0 +1,437 @@
+#!/bin/bash
+
```

Comment:
This doesn't looks like a script that really belongs in local/.  What changes did you make to Karel's standard script in steps/?  Can't you just do it via options to the standard script?